### PR TITLE
test: incorporate newer azure-devtools

### PR DESCRIPTION
**Pull request by yugangw-msft**
Some appservice tests involving binary payload can playback; also use simple decorator to enable large payload 
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [na] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [na] Each command and parameter has a meaningful description.
- [na] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/5693
This is an **experimental preview** for @microsoft users.

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py [hunk 1]
```diff
@@ -168,12 +168,8 @@ def test_vm_image_list_offers(self):
 
 class VMImageListPublishersScenarioTest(ScenarioTest):
 
+    @AllowLargeResponse()
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py]: I think there are several other instances where this is should be applied. Do you plan to update the remaining instances?

Comment (yugangw-msft) [reply src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py]: Done

File: src/azure-cli-testsdk/setup.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
     'jmespath',
     'mock',
     'vcrpy>=1.10.3',
-    'azure-devtools==0.5.2'
+    'azure-devtools>=0.5.4'
```
Comment (yugangw-msft) [inline src/azure-cli-testsdk/setup.py]: Once the PR is approved, I shall ask to publish out a new azure-devtools containing https://github.com/Azure/azure-python-devtools/pull/40, and remove the private wheel from this PR

**Pull request merged**